moped-editor/src/views/projects/projectView/ProjectComponents/utils/makeComponentData.js

```diff
@@ -178,7 +178,7 @@ export const getFeatureChangesFromComponentForm = (
   } else if (newSchoolBeaconKnackId) {
     if (
       previousSchoolBeacon &&
-      newSchoolBeaconKnackId !== previousSchoolBeacon.id
```

Comment:
good catch! 🙏 this is making me question why we didn't go with Knack id for signals too, but I think that `signal_id` has shown to be reliable for them.